addons/transifex/models/ir_translation.py

```diff
@@ -8,17 +8,58 @@
     from ConfigParser import ConfigParser
 from os.path import join as opj
 import os
+import re
 import werkzeug
 
-from odoo import models, fields
+from odoo import models, fields, _
 from odoo.modules.module import ad_paths
 
+REGEX_PLACEHOLDERS = re.compile("%(\(\w+\)|)([0+-]|)(\d+|\*|)[diouxXcrsa]|%(\(\w+\)|)([0+-]|)(\d+|\*|)(\.(\d+|\*|)|)[eEfFgG]|%%")
+REGEX_XML_TAGS = re.compile("</?(\w+)")
 
 class IrTranslation(models.Model):
 
     _inherit = 'ir.translation'
 
     transifex_url = fields.Char("Transifex URL", compute='_get_transifex_url')
+    is_valid = fields.Boolean(default=True)
+
+    def _compute_validity(self):
+        bad = self.browse()
+        for translation in self:
```

Comment:
Might make sense to use filtered() instead of hand-rolling it?

Reply:
isn't filtered the root of all evils?

Reply:
It's the root of lots of evils, but here it's pretty much what's done by hand.